packages/@okta/i18n/src/properties/login.properties

```diff
@@ -597,6 +597,9 @@ password.expiring.subtitle.specific = When password expires you will be locked o
 password.expired.custom.submit = Go to {0}
 password.expired.custom.subtitle = This password is set on another website. Click the button below to go there and set a new password.
 
+#Captcha
+hcaptcha.footer.label = This site is protected by hCaptcha and its <a href="https://hcaptcha.com/privacy" target="_blank">Privacy Policy</a> and <a href="https://hcaptcha.com/terms" target="_blank">Terms of Service</a> apply.
```

Comment:
We should try to avoid embedding HTML in our properties files, as it'll require us to request new translations whenever the link and/or element requires a change.

Instead, use this syntax:

```js
# login.properties
hcaptcha.footer.label = This site is protected by hCaptcha and its <$1>Privacy Policy</$1> and <$2>Terms of Service</$2> apply.

# CaptchaView.js
...
<span class="footer-text">
  {{i18n code="hcaptcha.footer.label" bundle="login" $1="<a href='https://hcaptcha.com/privacy' target='_blank'>$1</a>" $2="<a href='https://hcaptcha.com/terms' target='_blank'>$2</a>")}}
</span>
```

This way our view manages the template.

Reply:
Good idea, I'll take this approach instead.